### PR TITLE
refactor, chore, fix (Egen) : removes boilerplate and fixes notebook pytorch_train_deploy_models_with_prebuilt_containers.ipynotebook

### PR DESCRIPTION
**Pull request by k-root**
Following are the changes:
- changes `APP_NAME` as the dist generated is in canonical and gsutil and other commands are unable to find the original reference
- removes boilerplate
-  adds colab enterprise and reformats all the run buttons
- removes future tense references and gerands
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
       "outputs": [],
       "source": [
-        "# Copyright 2023 Google LLC\n",
+        "# Copyright 2024 Google LLC\n",
```
Comment (katiemn) [inline notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb:11]: The year can stay 2023

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb [hunk 2]
```diff
@@ -82,11 +87,11 @@
         "\n",
         "This tutorial uses the following Google Cloud ML services and resources:\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb:88]: Change "Google Cloud ML" to "Vertex AI"

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb [hunk 3]
```diff
@@ -82,11 +87,11 @@
...
         "\n",
-        "- `Vertex AI Training` service\n",
-        "- `Vertex AI Prediction` service\n",
-        "- `Vertex AI Model Registry`\n",
-        "- `Vertex AI Model` resources\n",
-        "- `Vertex AI Endpoint` resources\n",
+        "- Vertex AI Training service\n",
+        "- Vertex AI Prediction service\n",
+        "- Vertex AI Model Registry\n",
+        "- Vertex AI Model resources\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb:93]: Should be:
- Vertex AI model resources"

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb [hunk 4]
```diff
@@ -82,11 +87,11 @@
...
+        "- Vertex AI Endpoint resources\n",
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb:94]: Should be:
- Vertex AI endpoint resources

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb [hunk 5]
```diff
@@ -988,7 +922,7 @@
       "source": [
         "### Upload model for deployment\n",
         "\n",
-        "Next, you upload the model artifacts to `Vertex AI Model Registry`, which will create a `Vertex AI Model` resource for your model. This tutorial uses the PyTorch v1.11 container, but for your own use case, you can choose from the list of [PyTorch prebuilt containers](https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers#pytorch)."
+        "Next, you upload the model artifacts to `Vertex AI Model Registry`, which creates a `Vertex AI Model` resource for your model. This tutorial uses the PyTorch v1.11 container, but for your own use case, you can choose from the list of [PyTorch prebuilt containers](https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers#pytorch)."
```
Comment (kittyabs) [inline notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb:925]: remove back ticks and change "Model" to lower case:

- Vertex AI model

**Review by kittyabs — COMMENTED** [review]
Some suggested edits

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb (hunk 4 above)
Comment (k-root) [reply notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb]: pushed the change, but to confirm, should this actually be "Vertex AI Endpoint" itself as "Endpoint" is the name of a resource itself?

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb (hunk 3 above)
Comment (k-root) [reply notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb]: updated.

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb (hunk 2 above)
Comment (k-root) [reply notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb]: rectified

File: notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb (hunk 1 above)
Comment (k-root) [reply notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb]: rectified

**Review by katiemn — APPROVED** [review]

**Pull request merged**